static/app/views/issueDetails/traceTimeline/traceTimeline.tsx

```diff
@@ -49,28 +62,56 @@ export function TraceTimeline({event}: TraceTimelineProps) {
 
   return (
     <ErrorBoundary mini>
-      <TimelineWrapper>
-        <div ref={timelineRef}>
-          <TimelineEventsContainer>
-            <TimelineOutline />
-            {/* Sets a min width of 200 for testing */}
-            <TraceTimelineEvents event={event} width={Math.max(width, 200)} />
-          </TimelineEventsContainer>
-        </div>
-        <QuestionTooltipWrapper>
-          <QuestionTooltip
-            size="sm"
-            title={t(
-              'This is a trace timeline showing all related events happening upstream and downstream of this event'
-            )}
-            position="bottom"
-          />
-        </QuestionTooltipWrapper>
-      </TimelineWrapper>
+      {timelineStatus === 'shown' ? (
+        <TimelineWrapper>
```

Comment:
This `TimelineWrapper` block stays the same.

Using the Split view without whitespaces helps to see it more clearly:
<img width="249" alt="image" src="https://github.com/getsentry/sentry/assets/44410/436f07a0-6dc3-49cc-8aac-2e001cafbb59">